src/os.cpp

```diff
@@ -291,13 +291,39 @@ void os_path_resolve(Buf *ref_path, Buf *target_path, Buf *out_abs_path) {
     return;
 }
 
-int os_fetch_file(FILE *f, Buf *out_buf) {
+int os_fetch_file(FILE *f, Buf *out_buf, bool skip_shebang) {
```

Comment:
Had a look and to me it still seemed appropriate keep the file reading in `os.cpp`. I've just opted for a variable to specify if we should skip the shebang and adjusted uses accordingly.

Reply:
Sounds good :+1: